projects/github.com/nishanths/license/package.yml

```diff
@@ -20,7 +20,6 @@ build:
       LDFLAGS:
         - -buildmode=pie
   script: |
-    # cd ./cmd/xc
```

Comment:
intentional?

Reply:
Yeah it's leftover from my previous cleanup PR (#155), I figured it'd be okay to remove it here but I can open a separate PR if needed :)

Reply:
cool. This is fine, just wanted to check.